floodviz/map_utils.py

```diff
@@ -20,8 +38,19 @@ def site_dict(site_list, url_prefix):
     # get data from url
     req = requests.get(url)
 
+    if not req.text.startswith('#'):
+        print("Error: request returns invalid data")
+        return
+
     # put data into list
-    data = req.text.splitlines()[29:]
+    data = []
+    for line in req.text.splitlines():
+        if not line.startswith('#'):
```

Comment:
This will eliminate any lines that start with #. The rdb standard says that the first line without a # starts the file. It's possible that a data column might start with a #. This would erronously omit it. You should write this so once the first non comment line is found, the remaining lines are data.